easybuild/easyconfigs/a/Armadillo/Armadillo-7.950.1-intel-2016b-Python-2.7.12.eb

```diff
@@ -0,0 +1,23 @@
+name = 'Armadillo'
+version = '7.950.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://arma.sourceforge.net/'
+description = """Armadillo is an open-source C++ linear algebra library (matrix maths) aiming towards
+ a good balance between speed and ease of use. Integer, floating point and complex numbers are supported,
+ as well as a subset of trigonometric and statistics functions."""
+
+toolchain = {'name': 'intel', 'version': '2016b'}
+
+sources = [SOURCELOWER_TAR_XZ]
+source_urls = ['http://sourceforge.net/projects/arma/files']
```

Comment:
On Linux: `sha256sum path/to/source`; on macOS: `shasum -a 256 path/to/sum`

We have plans for adding support in EasyBuild itself for this (something like `eb --inject-checksum=sha256 foo.eb`), but it's no more than an idea for now...